helpers/requests/requests.go

```diff
@@ -0,0 +1,87 @@
+package requests
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// ConstructGetURL Constructs a URL to use in a GET request.
+// Queries are passed as a map of string arrays
```

Comment:
The function name is a word in the sentence. No need to capitalize the word "Constructs" here.

Also, this creates a kind of double-wording. Suggest using "creates" instead.

The second sentence is sort of implied from the function signature IMO.

```suggestion
// ConstructGetURL creates a GET request from a base URL, a map of query
// parameters, and a formatted path.
```